### PR TITLE
testing the update user route

### PR DESCRIPTION
**Pull request by iljaSL**

### CODE REVIEW

File: backend/tests/userUpdateTest.test.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  await User.deleteOne({
+    username: 'leon',
+  });
+
+  await request.post('/api/users/').send(userUtils.newValidUser);
+
+  user = await updateUtils.userInDb();
+});
+
+describe('Login User API Tests', () => {
```
Comment (lapaset) [inline backend/tests/userUpdateTest.test.js]: Login should be update?

File: backend/tests/userUpdateTest.test.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  test('login user success - expect 200', async () => {
```
Comment (lapaset) [inline backend/tests/userUpdateTest.test.js]: Is this test needed in this file?

**Review by lapaset — CHANGES_REQUESTED** [review]
Tests run beautifully and have nice coverage for our needs. Just one output issue and possible extra test pasted from login tests

File: backend/tests/userUpdateTest.test.js (hunk 1 above)
Comment (iljaSL) [reply backend/tests/userUpdateTest.test.js]: Forgot to change the test description, will change that.

File: backend/tests/userUpdateTest.test.js (hunk 2 above)
Comment (iljaSL) [reply backend/tests/userUpdateTest.test.js]: You are right, I can move it to the beforeAll function in order to extract the token, should't be a test in this case.

**Comment by iljaSL** [comment]
@lapaset new commit is up :)

**Pull request merged**